CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing to GBIF Pipelines
+
+The GBIF Pipelines development community welcomes contributions from anyone! Thank you.
+
+If you have questions please open [an issue](https://github.com/gbif/pipelines/issues/new) or contact the developers at informatics@gbif.org.
+
+There are different ways you can contribute that help this project:
+- Log [issues](https://github.com/gbif/pipelines/issues) and help document a bug or specify new functionality
+- Improve documentation
+- Provide code submissions that address issues or bring new functionalities
+- Help test the functionalities offered in the project
+- Help improve this guide
+
+## Contributing code
+
+Below is a tutorial for contributing code to Pipelines, covering our tools and typical process in detail.
+
+### Prerequisites
+
+To contribute code, you need
+
+- a GitHub account
+- a Linux, MacOS, or Microsoft Windows development environment with Java JDK 8 installed
+- Maven (version 3.6+)
+
```

Comment:
Maybe specify Java 8 version? I tried to use jdk 11 and target 8, and it doesn't work